src/sgx/faasm_sgx_enclave.cpp

```diff
@@ -417,30 +421,30 @@ extern "C"
             wasm_runtime_unload(tcs_ptr->module);
             _FAASM_SGX_TCS_FREE_SLOT(*thread_id);
             free(tcs_ptr);
-#if (FAASM_SGX_DEBUG)
-            ocall_printf(wamr_error_buffer);
-#endif
+            SGX_DEBUG_LOG(wamr_error_buffer);
             return FAASM_SGX_WAMR_MODULE_INSTANTIATION_FAILED;
         }
 
         return FAASM_SGX_SUCCESS;
     }
 
+    /* Initialise WebAssembly Micro Runtime inside the enclave.
+     *
+     * This method sets up the WAMR runtime, and initialises all enclave-related
+     * variables. Currently, this happens _once_ per Faasm instance. This is,
+     * we only run one enclave per Faasm instance.
+     */
     faasm_sgx_status_t faasm_sgx_enclave_init_wamr(void)
     {
-#if (FAASM_SGX_DEBUG)
-        // Set our ocall_printf as default printf function if FAASM_SGX_DEBUG ==
-        // 1
-        os_set_print_function((os_print_function_t)ocall_printf);
-#endif
+        os_set_print_function((os_print_function_t)SGX_DEBUG_LOG);
 
         // Initialize FAASM-SGX TCS
+        // Note - currently we just support single-threaded execution, thus
+        // only one TCS slot.
```

Comment:
If the value of `FAASM_SGX_INIT_TCS_SLOTS` was increased, would this comment become invalid? I feel like the one in `enclaveConfig.h` is probably enough.